sashimi-webapp/src/logic/documentPackager/conditionalProcessor.js

```diff
@@ -0,0 +1,240 @@
+/* Conditional processor to format what to show and what to hide in viewer
+** Referenced from markdown-it-inline-comments plugin
+** See here: https://github.com/jay-hodgson/markdown-it-inline-comments
+** Process =>hide 'This is a Text' <= as ''
+*/
+let fileName;
+
+function findName(state, startLine) {
+  const indent = state.sCount[startLine];
+  const end = state.eMarks[startLine];
+  const text = state.getLines(startLine, startLine + 1, indent, true).toString();
+  const parameter = text.slice(0, 9);
+  if (parameter === 'fileName:') {
+    fileName = text.slice(9, end);
+  }
+}
+
+function hideShowInline(state, silent) {
+  const max = state.posMax;
+  const start = state.pos;
+  let content;
+  let found;
+
+  if (silent) { return false; } // don't run any pairs in validation mode
+  if (start + 8 >= max) { return false; }
+
+  if (state.src.charCodeAt(start) !== 61 /* = */ ||
+      state.src.charCodeAt(start + 1) !== 62 /* > */ ||
+      state.src.charCodeAt(start + 2) !== 104 /* h */ ||
+      state.src.charCodeAt(start + 3) !== 105 /* i */ ||
+      state.src.charCodeAt(start + 4) !== 100 /* d */ ||
+      state.src.charCodeAt(start + 5) !== 101 /* e */
+      ) {
+    return false;
+  }
+  state.pos = start + 6;
+
+  // check for end in same line
+  while (state.pos + 2 <= max) {
+    if (state.src.charCodeAt(state.pos) === 60 /* < */ &&
+        state.src.charCodeAt(state.pos + 1) === 61 /* = */
+    ) {
+      found = true;
+      state.pos += 2;
+      break;
+    }
+
+    state.md.inline.skipToken(state);
+  }
+
+  if (!found || start + 1 === state.pos) {
+    state.pos = start;
+    return false;
+  }
+
+  content = state.src.slice(start + 6, state.pos);
+
+  // don't allow unescaped newlines inside
+  if (content.match(/(^|[^\\])(\\\\)*[\n]/)) {
+    state.pos = start;
+    return false;
+  }
+  return true;
+}
+
+/*
```

Comment:
Consider using jsdoc documentation format for function.
http://usejsdoc.org/about-getting-started.html

VSCode is able to display jsdoc documentation when other developer hover over the functions